okta/resource_user_test.go

```diff
@@ -254,6 +306,30 @@ resource "okta_user" "test_%s" {
 `, r, r, r)
 }
 
+func testOktaUserConfig_groupAssign(r string) string {
+	return fmt.Sprintf(`
+resource "okta_user" "test_acc_%s" {
+  first_name  = "TestAcc"
+  last_name   = "%s"
+  login       = "test-acc-%s@testing.com"
```

Comment:
Very minor but I see this string being built a lot. I think I added a function `buildResourceName` and `buildResourceFQN`. To counter myself, IDEs are good at search and replace so very minor, in fact probably not worth your time but it's there for the future.